test/TEST-21-DFUZZER/test.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+# SPDX-License-Identifier: LGPL-2.1-or-later
+set -e
+
+TEST_DESCRIPTION="Fuzz our D-Bus interfaces with dfuzzer"
+TEST_NO_NSPAWN=1
```

Comment:
I think it would be great if it was possible to run it in containers eventually.

In the meantime, I commented this out and ran the test in a container. `dfuzzer` failed to activate `org.freedesktop.network1` (as expected) but the container got stuck
```sh
[138852.713992] testsuite-21.sh[326]: Cannot open suppression file './dfuzzer.conf'
[138852.719082] testsuite-21.sh[326]: Loading suppressions from file '/etc/dfuzzer.conf'
[138852.719903] testsuite-21.sh[326]: [SESSION BUS]
[138852.728448] testsuite-21.sh[326]: Bus not found.
[138852.729512] testsuite-21.sh[326]: [SYSTEM BUS]
[138877.760142] testsuite-21.sh[326]: Error while activating 'org.freedesktop.network1': Timeout was reached.
[138877.763772] testsuite-21.sh[326]: Error while calling method 'GetConnectionUnixProcessID': GDBus.Error:org.freedesktop.DBus.Error.NameHasNoOwner: The connection does not exist
[138877.765029] testsuite-21.sh[326]: Couldn't get the PID of the tested process
[138877.765767] testsuite-21.sh[326]: Exit status: 4
[138877.799794] testsuite-21.sh[325]: Finished with result: exit-code
[138877.800325] testsuite-21.sh[325]: Main processes terminated with: code=exited/status=4
[138877.800901] testsuite-21.sh[325]: Service runtime: 25.089s
[138877.801466] testsuite-21.sh[325]: CPU time consumed: 28ms
[138877.833149] testsuite-21.sh[83]: + at_exit
[138877.833702] testsuite-21.sh[83]: + local found_cd=0
[138877.834067] testsuite-21.sh[83]: + read -r exe
[138877.836818] testsuite-21.sh[334]: ++ coredumpctl -F COREDUMP_EXE
[138877.838069] testsuite-21.sh[335]: ++ sort -u
[138877.912147] testsuite-21.sh[83]: + [[ 0 -eq 0 ]]
[138877.914151] testsuite-21.sh[83]: + journalctl --rotate --vacuum-size=16M
[138878.086090] testsuite-21.sh[83]: + [[ 1 -ne 0 ]]
[138878.086916] testsuite-21.sh[83]: + systemctl start end.service
```

Reply:
Looks like it can be "fixed" with `systemctl start --job-mode=isolate end.service` but I have to admit I haven't figured out why the test tries to start `end.service` in the first place.

Reply:
Got it. `dfuzzer` kicks `end.service` out with `ClearJobs` when it fuzzes `systemd`. Here it wasn't kicked out because `systemd` wasn't fuzzed so the test got stuck. I think `--job-mode=isolate` should do then.

Reply:
Nice catch, thanks!

Reply:
In the end I had to switch to `--job-mode=flush`, since `--job-mode=isolate` apparently requires a unit with `AllowIsolate=yes`, which introduced other issues. With `--job-mode=flush` it seems to work in nspawn as well (at least on my machine):

```
[4813392.164242] testsuite-21.sh[79]: + at_exit
[4813392.164595] testsuite-21.sh[79]: + local found_cd=0
[4813392.164835] testsuite-21.sh[79]: + read -r exe
[4813392.165568] testsuite-21.sh[318]: ++ coredumpctl -F COREDUMP_EXE
[4813392.166143] testsuite-21.sh[319]: ++ sort -u
[4813392.195955] testsuite-21.sh[79]: + [[ 0 -eq 0 ]]
[4813392.196240] testsuite-21.sh[79]: + [[ 1 -ne 0 ]]
[4813392.196418] testsuite-21.sh[79]: + systemctl start --job-mode=flush end.service
[4813392.240030] sh[322]: + systemctl poweroff --no-block
[4813392.261461] testsuite-21.sh[321]: Job for end.service canceled.
[  OK  ] Stopped testsuite-21.service.
[  OK  ] Stopped end.service.
```